src/ucs/async/async.h

```diff
@@ -90,20 +90,44 @@ static inline int ucs_async_check_miss(ucs_async_context_t *async)
     return 0;
 }
 
+/**
+ * Returns whether a context blocked or not.
+ *
+ * @param _async Event context to check `is_locked` status for.
```

Comment:
pls use one term `is_locking` or `is_blocking`
btw why do you use `block(ing)` thru the code?

Reply:
> pls use one term is_locking or is_blocking

done

> btw why do you use block(ing) thru the code?

because we have `UCX_ASYNC_BLOCK`/`UCS_ASYNC_UNBLOCK`